terraform/nomad-whisper/create-nomad-whisper-pool-stack.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
This should script be the whole thing, no need to have two create scripts.  The other directory had two because the lower script was called by multiple higher level scripts (nomad, whisper, general and x86) whereas this one should be all one script, no need for 2.